tests/unit/web/test_views.py

```diff
@@ -46,6 +46,35 @@ def test_create_contact_message_with_no_data(self):
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
 
+class CreateTeamMember(APITestCase):
+    def setUp(self):
+        self.url = reverse_lazy('web:our_team')
+        self.contributor = {
+            'name': 'John Snow',
```

Comment:
Can we rename the user to `Test User` and change other field values in similar way.